dashboard_documentation.md

```diff
@@ -69,6 +69,94 @@ Create New Site
    <img class="my-3" src="/images/plant_site.png"/>
 
 
+Create New Aggregate
+--------------------
+{: .anchor}
+
+Aggregates are created by first defining metadata for the aggregate, then
+associating existing observations with it. The aggregate metadata determines
+how aggregation will be performed and the characteristics of the resulting
+timeseries data.
+
+Observations are included in an aggregate between an *Effective From* and
```

Comment:
This paragraph seems out of place because these options are not exposed in the aggregate creation form. Move to Add Observations to an Aggregate?